packages/jest-changed-files/src/git.js

```diff
@@ -15,10 +15,10 @@ import type {Path} from 'types/Config';
 const path = require('path');
 const childProcess = require('child_process');
 
-type Options = { 
+type Options = {|
```

Comment:
hm.. i can't build jest locally because babel freaks out here. where can i read about this syntax?